test/vars/CloudSpec.groovy

```diff
@@ -426,6 +426,15 @@ class CloudSpec extends JenkinsPipelineSpecification {
         1 * getPipelineMock("sh")("docker pull --platform PLATFORM IMAGE")
     }
 
+    def "[cloud.groovy] dockerBuildPlatformImage outputToFile"() {
+        when:
+        groovyScript.dockerBuildPlatformImage('IMAGE', 'PLATFORM', true)
+        then:
+        1 * getPipelineMock("sh")("docker buildx build --push --sbom=false --provenance=false --platform PLATFORM -t IMAGE . > IMAGE-PLATFORM-build.log")
```

Comment:
Just tested this, we need to replace all slashes at least, otherwise we're getting non-existent directory error.

Reply:
And also we should take care where we put the logs. Given that this is jenkins library and we know we might be running in docker container, we can put into workspace.

Reply:
And I am commenting on test, not impl, wonderful. I'll add a suggestion in the right place.